ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -641,23 +641,30 @@ def test_numpy_negative_binomial(
 @handle_frontend_test(
     fn_tree="numpy.random.noncentral_chisquare",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
-        num_arrays=2,
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_value=0,
+        exclude_min=True,
+    ),
+    dtype_and_y=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
         min_value=0,
     ),
     size=helpers.get_shape(allow_none=True),
     test_with_out=st.just(False),
 )
 def test_numpy_noncentral_chisquare(
     dtype_and_x,
+    dtype_and_y,
```

Comment:
Why don't you use `df` and `nonc` instead of `x` and `y`? I feel it would be more readable